nfpm.go

```diff
@@ -160,14 +181,42 @@ type RPM struct {
 	Compression string `yaml:"compression,omitempty"`
 	// https://www.cl.cam.ac.uk/~jw35/docs/rpm_config.html
 	ConfigNoReplaceFiles map[string]string `yaml:"config_noreplace_files,omitempty"`
+	Signature            RPMSignature      `yaml:"signature,omitempty"`
+}
+
+type RPMSignature struct {
+	// PGP secret key, can be ASCII-armored
+	KeyFile       string `yaml:"key_file,omitempty"`
+	KeyPassphrase string // populated from environment variable
+}
+
+type APK struct {
+	Signature APKSignature `yaml:"signature,omitempty"`
+}
+
+type APKSignature struct {
+	// RSA private key in PEM format
+	KeyFile       string `yaml:"key_file,omitempty"`
+	KeyPassphrase string // populated from environment variable
+	// defaults to <maintainer email>.rsa.pub
+	KeyName string `yaml:"key_name,omitempty"`
 }
 
 // Deb is custom configs that are only available on deb packages.
 type Deb struct {
-	Scripts         DebScripts  `yaml:"scripts,omitempty"`
-	Triggers        DebTriggers `yaml:"triggers,omitempty"`
-	Breaks          []string    `yaml:"breaks,omitempty"`
-	VersionMetadata string      `yaml:"metadata,omitempty"` // Deprecated: Moved to Info
+	Scripts         DebScripts   `yaml:"scripts,omitempty"`
+	Triggers        DebTriggers  `yaml:"triggers,omitempty"`
+	Breaks          []string     `yaml:"breaks,omitempty"`
+	VersionMetadata string       `yaml:"metadata,omitempty"` // Deprecated: Moved to Info
+	Signature       DebSignature `yaml:"signature,omitempty"`
+}
+
+type DebSignature struct {
+	// PGP secret key, can be ASCII-armored
+	KeyFile       string `yaml:"key_file,omitempty"`
+	KeyPassphrase string // populated from environment variable
```

Comment:
maybe 
```suggestion
	KeyPassphrase string `yaml:"-"` // populated from environment variable
```
to make sure its not populated from yaml?

Reply:
Oh, yes I forgot the ignore tag